src/filter/component.js

```diff
@@ -294,7 +325,7 @@ ngeo.FilterController = class {
 
 
   /**
-   * @param {!ngeo.FilterController.Condition} condition Condition to set.
+   * @param {!ngeo.filter.component.Condition} condition Condition to set.
```

Comment:
It's not better to have a ngeox.filterCondition ?

Reply:
Moved it.